sklearn/utils/estimator_checks.py

```diff
@@ -1145,6 +1146,42 @@ def check_classifiers_train(name, Classifier):
                 assert_array_equal(np.argsort(y_log_prob), np.argsort(y_prob))
 
 
+def check_classifiers_multilabel_representation_invariance(name, Classifier):
+
+    if name not in MULTI_OUTPUT:
+        raise SkipTest
+
+    X, y = make_multilabel_classification(n_samples=100,
+                                          n_features=20,
+                                          n_classes=5,
```

Comment:
nitpick: can you club some lines together... This takes up more lines than is needed...